package-lock.json

```diff
@@ -0,0 +1,5339 @@
+{
```

Comment:
What is this file?

Reply:
I think it was created when I was checking whether node was installed, but I deleted it.